src/dsync/dsync.c

```diff
@@ -2968,6 +2971,20 @@ static int dsync_daos(
 }
 #endif
 
+static bool is_mtime_nsec_used(char *srcpath, char *dstpath)
+{
+    struct stat srcpath_stat, dstpath_stat;
+    if (stat(srcpath, &srcpath_stat) == 0 && stat(dstpath, &dstpath_stat) == 0) {
+        if (srcpath_stat.st_mtim.tv_nsec == 0 || dstpath_stat.st_mtim.tv_nsec == 0) {
+           return false; 
+        } else {
+            return true;
+        }
+    } else {
+        return false;
```

Comment:
If we can't stat() one of src and dest, then we don't know whether nsec are supported.  

Is the "safe" thing to return true, so that we compare nsec?  Then we might copy more files than we need to, but we can be sure the dest and the src match.

Reply:
ok, will return true. Make sense, I think!